client/my-sites/exporter/notices.jsx

```diff
@@ -62,26 +63,8 @@ class Notices extends Component {
 	}
 
 	guidedTransferNotice() {
-		const {
-			translate,
-			siteSlug
-		} = this.props;
-
 		if ( this.props.isGuidedTransferAwaitingPurchase ) {
-			return (
-				<Notice
-					status="is-warning"
-					showDismiss={ false }
-					text={ translate(
-						"It looks like you've started a Guided Transfer. " +
-						"We just need your payment to confirm the transfer and " +
-						"then we'll get started!" ) }
-				>
-					<NoticeAction href={ `/settings/export/guided/${ siteSlug }` }>
-						{ translate( 'Continue' ) }
-					</NoticeAction>
-				</Notice>
-			);
+			return <CompletePurchaseNotice />;
 		}
 
 		return null;
```

Comment:
Not related to this pull request necessarily, but a few thoughts:

- Your function could have been indented one less for improved readability by returning early for the `! this.props.isGuidedTransferAwaitingPurchase` case.
- You could remove this `return` and the function would work just as well, because the inferred `return undefined;` can be handled by React
- The logic of this function is simple enough that you may as well inline it into the render function:

```js
const { isGuidedTransferAwaitingPurchase } = this.props;

return (
	{ this.exportNotice() }
	{ isGuidedTransferAwaitingPurchase && <CompletePurchaseNotice /> }
);
```